doc/release-notes/0.35/0.35.md

```diff
@@ -32,6 +32,8 @@ OpenJ9 release 0.35.0 supports OpenJDK 8, 11, 17.
 
 All releases are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests provided by Adoptium.
 
+AArch64 macOS&reg; is now a fully supported, production-ready target for OpenJDK 17.
```

Comment:
It's supported for OpenJDK 11 and later, which means 11 and 17 in the release, but also 19/20/etc when these versions are released later.